cybereason/help.md

```diff
@@ -39,6 +39,53 @@ Example input:
 
 ### Actions
 
+#### Isolate Machine
+
+This action is used to isolate a machine associated with the root cause of a Malop, or to remediate a process not involved in a malop.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|actions_by_machine|object|None|False|Actions by machine|None|{"126811122.2298225282553311122": [{"targetId": "531122333.-3391199199911692223","actionType": "KILL_PROCESS"}]}|
```

Comment:
Can we make this easier for the user? Can this object be split into separate inputs?